src/components/main.js

```diff
@@ -577,7 +577,7 @@ class Main extends React.Component {
             <SettingOutlined/>
             <span>Settings</span>
           </Menu.Item>
-          <Menu.SubMenu title="Wallet" key="submenu-wallet">
+          <Menu.SubMenu title="BTC Wallet" key="submenu-wallet">
```

Comment:
I'd rather not do this even though I understand why you would suggest it. This would put "BTC" on the main menu. Right now the wallet submenu is collapsed by default. We could rename the span on L583 to `BTC Wallet` instead though.